browser-test/src/support/admin_programs.ts

```diff
@@ -1337,4 +1337,16 @@ export class AdminPrograms {
     )
     expect(sentEmail.Body.text_part).toContain(emailBody)
   }
+
+  async seedProgramsAndCategories() {
```

Comment:
(suggestion) If you have a specific reason to do this here this is fine, but if not can you swap to the [seedPrograms in index.ts](https://github.com/civiform/civiform/blob/a398fb2a0c4704570f522a399b75197853cbc197/browser-test/src/support/index.ts#L258).

Reply:
Done.  Also, changed the name of `seedPrograms` to `seedProgramsAndCategories`.